### ISSUE TITLE
Fix false positives for single value within template literal

### ISSUE THREAD

**Issue by jdvivar** [stylelint/stylelint#4655]
I've looked for similar bugs and although there are similar titles I couldn't find this one. It's not in the changelog and it's reproducible in the [playground demo](https://stylelint.io/demo). I also created this small repo to more easily reproduce it: https://github.com/jdvivar/demo-bug-stylelint

> Clearly describe the bug

False positive on single css-in-js value

> Which rule, if any, is the bug related to?

Not a rule, but a syntax error

> What code is needed to reproduce the bug?

with this `test.ts` file:
```js
import { css } from 'lit-element'
export const max = css`100px`
```

> What stylelint configuration is needed to reproduce the bug?

and this config `.stylelintrc.json`:
```json
{
  "extends": "stylelint-config-standard"
}
```

> Which version of stylelint are you using?

`13.2.1`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI with `stylelint --syntax css-in-js "test.ts"`
CLI with `stylelint "test.ts"`

Both with the same output

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Yep, CSS-in-JS

> What did you expect to happen?

Nothing to be flagged

> What actually happened (e.g. what warnings or errors did you get)?

```bash
test.ts
 2:24  ✖  Unknown word   CssSyntaxError
```

**Comment by jeddy3** [comment]
@jdvivar Thanks for the clear report, reproducible demo and repository!

It's likely a problem with [the parser stylelint uses to support CSS-in-JS](https://github.com/stylelint/postcss-css-in-js). I've added this issue to https://github.com/stylelint/stylelint/issues/4574, which is a growing list of the problems with the syntax that we need help to fix.

Please consider [contributing to the parser](https://stylelint.io/developer-guide/syntaxes#working-on-syntaxes) if you have time.

**Comment by jeddy3** [comment]
Closing as the [`syntax` option was removed](https://github.com/stylelint/stylelint/issues/5289) for the [14.0.0 release](https://github.com/stylelint/stylelint/issues/5205). (See the [migration guide](https://github.com/stylelint/stylelint/blob/main/docs/migration-guide/to-14.md#syntax-option-and-automatic-inferral-of-syntax).)

We'll also [be deprecating](https://github.com/stylelint/postcss-css-in-js/issues/225) our [forked CSS-in-JS package](https://github.com/stylelint/postcss-css-in-js), in favour of leaner custom syntaxes.

Please consider [writing a custom syntax](https://github.com/stylelint/stylelint/blob/main/docs/developer-guide/syntaxes.md) specific to the CSS-in-JS library in this issue and fixing the bug there.
